docs/resources/split_tunnel.md

```diff
@@ -35,7 +35,7 @@ resource "cloudflare_split_tunnel" "example_split_tunnel_include" {
 }
 
 # Create a device policy
-resource "cloudflare_device_policy" "developer_warp_policy" {
```

Comment:
thanks for the PR! unfortunately, the file you've edited is a generated artifact of the documentation process. to see how to make changes to the documentation, you're best to check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md